components/partials/header.js

```diff
@@ -44,57 +44,62 @@ export default function Header() {
               </Link>
             </div>
 
-            <div className="flex grow items-center">
-              <ul className="flex flex-row flex-wrap items-center justify-start mb-[10px] my-0 md:ml-auto p-0 text-black-900/80">
-                <li className="ml-0 my-2 sm:my-0 p-0">
+            <div className="flex grow items-center mt-2 lg:mt-0">
+              <ul className="flex flex-row flex-wrap items-center justify-start lg:ml-auto text-[1rem] md:text-[1.09375rem] lg:text-[1.1875rem] leading-[1.125rem] md:leading-[1.28125rem] lg:leading-[1.4375rem]">
+                <li className="ml-0 my-2 sm:my-0">
                   <Link
-                    href="/"
-                    className={`relative mr-[20px] sm:mr-[40px] text-[1rem] md:text-[1.0625rem] lg:text-[1.125rem] font-normal leading-[1.125rem] md:leading-[1.5rem] lg:leading-[1.875rem] tracking-[1px] after:absolute after:top-[29px] after:md:top-[24px] after:bottom-0 after:left-0 after:w-2/4 after:h-[2px] after:bg-black-900 ${
-                      router.pathname == "/"
-                        ? ""
-                        : "after:origin-bottom-left after:duration-300 after:scale-x-0 hover:after:scale-x-100 hover:after:origin-bottom-left"
+                    href="https://canopas.com/portfolio"
```

Comment:
Let's make `https://canopas.com` constant. In case we need to change it as per requirement, we don't need to change it everywhere.